capstone/scripts/ingest_tt_data.py

```diff
@@ -91,5 +92,20 @@ def copyModel(source, destination, field_map, dupcheck, dupe_field='id'):
 
         try:
             destination_record.save()
+            if source.__name__ == 'Reporters':
+                destination_record.jurisdictions.add(Jurisdiction.objects.get(name=source_record.state))
+                destination_record.save()
+
         except IntegrityError as e:
             print(e)
+
+def populate_jurisdiction():
+    """This populates the jurisdiction table based on what's in the tracking tool stub"""
+    reporters = Reporters.objects.order_by().values('state').distinct() 
+    for jurisdiction in reporters:
+        if Jurisdiction.objects.filter(name=jurisdiction['state']).count() > 1:
```

Comment:
Does this mean we'll create a maximum of two jurisdictions with a given state? If so maybe add a comment to explain the goal